cvat/apps/engine/views.py

```diff
@@ -928,11 +1128,58 @@ def upload_finished(self, request):
                         status=status.HTTP_400_BAD_REQUEST)
 
     @extend_schema(methods=['POST'],
-        summary='Method permanently attaches images or video to a task. Supports tus uploads, see more https://tus.io/',
+        summary="Method permanently attaches data (images, video etc.) to a task",
+        description=textwrap.dedent("""\
+            Allows to upload data to a task.
+            Supports the TUS open file uploading protocol (https://tus.io/).
+
+            Implements the following protocols:
+            a. A single Data request
+            b.1. An Upload-Start request
+            b.2.a. The regular TUS protocol requests (Upload-Length + Chunks)
+            b.2.b. Upload-Multiple requests
+            b.3. An Upload-Finished request
```

Comment:
It's not clear from the description that some of the list levels represent alternatives, while some represent sequential requests. I think it would be more instructive to spell it out, e.g.:

-----
To use this endpoint, send either a single Data request, or the following sequence of requests:

1. An Upload-Start request.
2. One or more of Upload-Length or Upload-Multiple request. For each `Upload-Length` request send, complete the upload using one or more `Chunk` requests.
3. An Upload-Finish request.
-----

It might also be worth mentioning that `Chunk` requests are send not to this endpoint, but to a file-specific one.